### DISCUSSION TITLE
站点数据统计希望能加上当天到目前为止的上传下载增量

### DISCUSSION THREAD

**Discussion by sn066** [jxxghp/MoviePilot#3425] [讨论]
### 当前程序版本

最新

### 运行环境

Docker

### 功能改进类型

插件

### 功能改进

希望站点数据统计插件增加一个统计类型，当天增量，统计数据为当天到当前时间为止各个站点的上传下载增量，就是原先nastool时的统计方式

### 参考资料

无

**Comment by hoey94** [comment]
对于上传和下载的今日增量这个功能已经实现了。
<img width="1301" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/13510799/9ecb3d03-6fb7-4968-9896-384fb322179d">

**Comment by sn066** [comment]
> 对于上传和下载的今日增量这个功能已经实现了。
> <img width="1301" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/13510799/9ecb3d03-6fb7-4968-9896-384fb322179d">
> 

我现在看消息通知的内容还不是，是这部分我还有缓存数据吗？
